uaclient/cli.py

```diff
@@ -325,10 +313,26 @@ def action_attach(args, cfg):
 
 
 def get_parser():
+    service_line_tmpl = " - {name}: {description}{url}"
+    description = [__doc__]
```

Comment:
A nit, but `description_lines` is often used to indicate this pattern is being used.